web_src/less/_base.less

```diff
@@ -340,8 +340,7 @@ a.label,
 .ui.search .results a,
 .ui .menu a,
 .ui.cards a.card,
-.issue-keyword a,
-a.commit-statuses-trigger {
```

Comment:
Why was the line removed?
Instead of being updated to

```css
a[data-trigger="commit-statuses"],
span[data-trigger="commit-statuses"] {
```

or similar…

Reply:
It has been replaced by the `gt-tdn` class which does the very same thing.

Reply:
(Only relevant for the `a` tag)